n3fit/src/n3fit/model_trainer.py

```diff
@@ -143,15 +142,13 @@ def __init__(
                 list with the replicas ids to be fitted
         """
         # Save all input information
-        self.exp_info = exp_info
-        if pos_info is None:
-            pos_info = []
+        self.exp_info = list(exp_info)
```

Comment:
Why is the `pos_info is None` conditional removed from here? It should be as possible as `integ_info` to get it as `None`

(actually maybe it should be, the code below,

```python
self.all_info = self.exp_info[0]
if pos_info is not None:
    self.all_info += pos_info
if integ_info is not None:
    self.all_info += integ_info

```

Reply:
This must have been an incorrect refactoring or rebase, we should indeed check for pos_info to be not None, and I think the cleanest way is to assign empty lists to the integ_info and pos_info class members whenever these arguments are None, and remove all the checks for None in the code below

Reply:
Done in rev. db24f63